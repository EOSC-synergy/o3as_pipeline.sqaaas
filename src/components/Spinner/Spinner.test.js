import { Spinner } from "./Spinner"; 
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

describe("testing spinner rendering", () => {
    it('renders without crashing', () => {
        render(<Spinner />);
    });

    it('renders OK', () => {
        const { container } = render(<Spinner />);
        expect(container).toMatchSnapshot();
    });

    it('displays the input text', () => {
        const { container } = render(<Spinner text={"loading"}/>)
        expect(container).toHaveTextContent("loading");
    });

});