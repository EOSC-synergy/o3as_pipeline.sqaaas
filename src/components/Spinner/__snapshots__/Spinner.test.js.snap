// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing spinner rendering renders OK 1`] = `
<div>
  <div
    class="spinner"
  >
    <div
      class="loader"
      style="height: 5em; width: 5em;"
    />
  </div>
</div>
`;
